### ISSUE TITLE
Upgrade path from 5.5.* to 5.6.*

### ISSUE THREAD

**Issue by jradtilbrook** [DarkaOnLine/L5-Swagger#232]
- L5-Swagger Version: 5.5.8.3
- PHP Version (`php -v`): 7.3.10
- OS: Arch linux

### Description:
*TLDR:* I don't think there is an upgrade path for 5.5.* to 5.6.*. But I could be doing something wrong.

I am trying to do a round of package updates and trying to update Laravel framework version from 5.5 to 5.6. I'm not very experienced with composer other than simple installs so I am having issues. I am unable to update either this package or laravel/framework at one time because they both need updating at the same time to stay inline with version constraints.  
Would you be open to adding a patch update to the `5.5.*` tag that would change the requirements on laravel/framework to something like `5.5.*||5.6.*`?

### Steps To Reproduce:
I have tried  `composer require 'laravel/framework:^5.6' 'darkaonline/l5-swagger:^5.6'` but it fails reporting constraints.

**Comment by DarkaOnLine** [comment]
You just need to change versions in your composer.json file and then run `composer update`. You do not need to require them again.

**Comment by jradtilbrook** [comment]
Okay that seems to have gotten further and run into conflicts in other packages. Thanks for the reply :+1:
